packages/app/src/styles/theme/mixins/_list-group.scss

```diff
@@ -49,7 +49,8 @@
       }
     }
 
-    &.grw-pagetree-current-page-item {
+    &.grw-pagetree-current-page-item,
+    &.grw-foldertree-current-folder-item {
```

Comment:
Is this `grw-foldertree-current-folder-item` class used?

Reply:
This class is no longer used, so I removed it